architecture-building-systems/CityEnergyAnalyst
# 3124 buildings with central AC getting cooled only by ARU

This PR solves issue #3124 by adding a function (`verify_hvac_system_combination`) to check whether an infeasible combination of systems (i.e., `HVAC_COOLING_AS3` or `HVAC_COOLING_AS4` with no mechanical ventilation) was selected.

If a building is found to have such an infeasible combination, a warning is printed to warn users of this issue, and a default ventilation system (`HVAC_VENTILATION_AS1`) is selected for the building in question.

This ensures that central and hybrid AC systems are operated correctly regardless of whether users select a ventilation system for the building or not.

To test:
1-Create a case study
2-Assign a building cooling system `HVAC_COOLING_AS3` or `HVAC_COOLING_AS4`, and ventilation system `HVAC_VENTILATION_AS0`
3-Run demand simulation
4-Verify whether the desired warning is printed and whether the systems are correctly activated in the simulation results.
